learning-code-examples/Cargo.toml

```
@@ -0,0 +1,62 @@
+[package]
+name = "learning-code-examples"
+version = "0.1.0"
+edition = "2021"
```

```suggestion
name = "learning-code-examples"
edition = "2021"
publish = false
```

The version number can be removed, since version tracking doesn't really make sense in this scenario. I also like to add `publish = false` to protect against accidental `cargo publish` calls, though this mistake is quite rare.

Additionally, it may be worth adding [`autoexamples = false`](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#target-auto-discovery). This will prevent example auto-discovery, protecting against confusing behavior where some examples are caught and some aren't.